frown

```diff
@@ -0,0 +1,14 @@
+Sad by Joan Stark
+
+     .-""""""-.
+   .'          '.
+  /   O      O   \
+ :           `    :
+ |                |   sad
+ :    .------.    :
+  \  '        '  /
+   '.          .'
+jgs  '-......-'
+
+
+(taken from https://www.asciiart.eu/computers/smileys)
```

Comment:
```
  , ; ,   .-'"""'-.   , ; ,
  \\|/  .'         '.  \|//
   \-;-/   ()   ()   \-;-/
   // ;               ; \\
  //__; :.         .; ;__\\
 `-----\'.'-.....-'.'/-----'
        '.'.-.-,_.'.'
jgs       '(  (..-'
            '-'
``` (same [source](https://www.asciiart.eu/computers/smileys))

Testing a fix for https://github.com/nix-community/infra/pull/30#issuecomment-645673503.

Reply:
\o/